opencog/nlp/relex2logic/rule-helpers.scm

```diff
@@ -466,26 +466,26 @@
 	(define adv (cog-name adv-node))
 	(define adv_instance (cog-name adv-instance-node))
 	(ListLink
-		(InheritanceLink  (ConceptNode adv_instance) (ConceptNode adv))
-			(ImplicationLink  (PredicateNode instance) (PredicateNode verb))
-			(r2l-wordinst-Predicate instance)
-			(r2l-wordinst-concept adv_instance)
-			(InheritanceLink  (SatisfyingSetLink (PredicateNode instance)) (ConceptNode adv_instance))
+		(ImplicationLink  (PredicateNode adv_instance) (PredicateNode adv))
+		(InheritanceLink  (ConceptNode instance) (ConceptNode verb))
```

Comment:
From a PLN perspective, it doesn't matter whether verbs are predicates or concepts, right? I suppose it does make some sense from an NLP viewpoint (I'm really unfamiliar with NLP in general).

So the problem is that rules like `advmod-rule` should not be used with nouns, I guess something is wrong in either the R2L rules or the LG. I'll look into that, obviously if you know how to fix it, let me know.